carma_cooperative_perception/src/multiple_object_tracker_component.cpp

```diff
@@ -155,6 +193,8 @@ static auto to_ros_msg(const mot::CtraTrack & track) noexcept
 
   msg.accel.accel.linear.x = mot::remove_units(track.state.acceleration);
 
+  msg.semantic_class = semantic_class_to_numeric_value(mot::get_semantic_class(track));
+
```

Comment:
Looks like we need a function in the library? `get_semantic_class`

Reply:
Yes. That function was introduced in a spike branch for `multiple_object_tracking`. We should merge those changes before merging this PR.